internal/model/openshift_objects.go

```diff
@@ -10,39 +10,50 @@ import (
 //OpenShift related structs
 
 const (
-	JenkinsIdled    = 0
+	// JenkinsIdled : Jenkins is not running (idle)
+	JenkinsIdled = 0
+	// JenkinsStarting : Jenkins is about to start
 	JenkinsStarting = 1
-	JenkinsRunning  = 2
+	// JenkinsRunning : Jenkins is running
+	JenkinsRunning = 2
 )
 
+// Object : Build Object
 type Object struct {
 	Type   string `json:"type"`
 	Object Build  `json:"object"`
 }
 
+// DCObject is DeploymentConfig Object
 type DCObject struct {
```

Comment:
This one too
DeploymentConfigObject ?

Reply:
+1

Reply:
You need to check whether this rename would break anything in terms of JSON marshalling though. I don't think so, but worth double checking